<launch>
	<arg name="num" default="0" />                                                    <!--POS:    X  Y  Z    ROT: Z     Y    X     PARENT       CHILD     -->
	<node pkg="tf2_ros" type="static_transform_publisher" name="aruco_camera_broadcaster" args="  0  0  0        3.14   0    0     camera       camera_aruco" />
	<node pkg="tf2_ros" type="static_transform_publisher" name="gTt_base_broadcaster"     args="  0  0 0.5       1.57 1.57   0     basic_shapes base_goal"/>
	<node pkg="tf2_ros" type="static_transform_publisher" name="goal_broadcaster"         args="  0  0  0          0    0    0     base_goal    new_nav_goal" />

	<node name="opencv_camera"     pkg="mycobot_280"       type="opencv_camera" args="$(arg num)"/>
	<node name="agv_detect_marker" pkg="camera_navigation" type="agv_detect_marker.py"/>
	
	<!--
	<node name="agv_publish_nav_goal" pkg="camera_navigation" type="agv_publish_nav_goal.py" />
	-->
</launch>